### ISSUE TITLE
Req: More detail or example for usage

### ISSUE THREAD

**Issue by itmanvn** [pingpong-labs/admin#30]
Hi OP,

This would be great if you put an example on how to use or more detail about usage.

Thank you from a guy new to L5.

**Comment by gravitano** [comment]
Yes, I haven't updated this wiki. Maybe I would create some video screencast or just update this wiki.

**Comment by adangbryen** [comment]
It is a good package, just read the code you will know how to use it.
maybe the doc needs more infomation.

**Comment by gravitano** [comment]
I am working on the documentation :)

**Comment by adangbryen** [comment]
need some help?

**Comment by gravitano** [comment]
If you want to help me, you can try creating or updating the wiki of this
repo. help is welcome and greatly appreciated.

**Comment by syahzul** [comment]
I have create a page for creating admin Role and User here:
[Admin Role and User](https://github.com/pingpong-labs/admin/wiki/Admin-Role-and-User)

**Comment by gravitano** [comment]
@syahzul Thank you :)
